resources/js/app.js

```diff
@@ -48,7 +48,7 @@ Vue.component(
 	  "user-dashboard-infrastructure",
 	  require("./components/Dashboard/UserDahboardInfrastructure.vue").default
 );
-Vue.component("user-dashboard-invoice", require("./components/Dashboard/UserDahboardInvocie.vue").default);
+Vue.component("user-dashboard-invoice", require("./components/Dashboard/UserDahboardInvoice.vue").default);
```

Comment:
can we correct the file name?
```suggestion
Vue.component("user-dashboard-invoice", require("./components/Dashboard/UserDashboardInvoice.vue").default);
```